.github/workflows/ci.yml

```diff
@@ -130,6 +130,48 @@ jobs:
         run: poetry run ./manage.py migrate --noinput
       - name: Test
         run: make test
+      - name: Upload coverage data
+        uses: actions/upload-artifact@v4
+        with:
+          name: coverage-data
+          path: .coverage
+          if-no-files-found: ignore
+          include-hidden-files: true
+
+  coverage:
```

Comment:
Question: Why is this a separate step rather than being handled in the `test-python` step?e

Reply:
A personal preference and habit from package work